playwright/tests/smoke/componentPicker.spec.ts

```diff
@@ -0,0 +1,197 @@
+/**
+ * Copyright © 2021 Johnson & Johnson
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+// componentPicker.spec.ts
+import { expect, Page, test } from '@playwright/test';
+import { FlowContainerPage } from '../../pages/flow-container-page';
+
+const NUMBER_OF_COMPONENTS = 26;
+
+async function checkAllComponentCheckboxes(page: Page, flowContainerPage: FlowContainerPage) {
+  expect(await page.locator(flowContainerPage.component.richText).getAttribute('disabled')).toBeFalsy();
+  expect(await page.locator(flowContainerPage.component.richText).getAttribute('disabled')).toBeFalsy();
+  expect(await page.locator(flowContainerPage.component.table).getAttribute('disabled')).toBeFalsy();
+  expect(await page.locator(flowContainerPage.component.image).getAttribute('disabled')).toBeFalsy();
+  expect(await page.locator(flowContainerPage.component.card).getAttribute('disabled')).toBeFalsy();
+  expect(await page.locator(flowContainerPage.component.contentful).getAttribute('disabled')).toBeFalsy();
+  expect(await page.locator(flowContainerPage.component.accordion).getAttribute('disabled')).toBeFalsy();
+  expect(await page.locator(flowContainerPage.component.list).getAttribute('disabled')).toBeFalsy();
+  expect(await page.locator(flowContainerPage.component.iframe).getAttribute('disabled')).toBeFalsy();
+  expect(await page.locator(flowContainerPage.component.socialShare).getAttribute('disabled')).toBeFalsy();
+  expect(await page.locator(flowContainerPage.component.youTube).getAttribute('disabled')).toBeFalsy();
+  expect(await page.locator(flowContainerPage.component.carousel).getAttribute('disabled')).toBeFalsy();
+  expect(await page.locator(flowContainerPage.component.curator).getAttribute('disabled')).toBeFalsy();
+  expect(await page.locator(flowContainerPage.component.flowContainer).getAttribute('disabled')).toBeFalsy();
+}
+
+test.describe('Component picker', async () => {
+  let page: Page;
+  test.beforeEach(async ({ browser }) => {
+    const context = await browser.newContext();
+    page = await context.newPage();
+    await page.goto('/flow-container');
+  });
+
+  test.afterEach(async () => {
+    await page.close();
+  });
+
+  test('Component picker: 1 - checking presence of all components by amount', async () => {
```

Comment:
@pomi ,

please split to a smaller test by a common sense depending on what is being tested